src/components/_app/PostHogProvider.tsx

```diff
@@ -34,7 +35,8 @@ const PostHogContext = createContext<{
 const CustomPostHogProvider = ({
   children,
 }: PropsWithChildren<unknown>): JSX.Element => {
-  const { id, addresses } = useUser()
+  const { account } = useWeb3React()
```

Comment:
`useUserPublic` doesn't contain the user address, so we decided we'll just use the address from web3-react here